### PR TITLE
[Javascript] Add naturaljs

### PR DESCRIPTION
**Pull request by jesusvilla**
Add naturaljs #3301

### CODE REVIEW

**Review by waghanza — CHANGES_REQUESTED** [review]

File: javascript/naturaljs-router/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "natural-framework": "^0.1.1"
```
[the remark this answers is not in the document]
Comment (jesusvilla) [reply javascript/naturaljs-router/package.json]: done

File: javascript/naturaljs-router/app.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    // type: 'uws', // 'uws' or 'node'
+    /* ssl: {
+      key: path.join(__dirname, './security/cert.key'),
+      cert: path.join(__dirname, './security/cert.pem')
+    } */
+  })
+  try {
+    createRoutes(router)
+    const port = await router.listen(3000)
+    console.log(`Listen http://localhost:${port}`)
```
[the remark this answers is not in the document]
Comment (jesusvilla) [reply javascript/naturaljs-router/app.js]: @waghanza done

**Comment by jesusvilla** [comment]
@waghanza, I already made the changes you requested

**Review by waghanza — APPROVED** [review]
Thanks for your contribution. I can give you any date for fresh results, but be sure I'm working on it ❤️

**Pull request merged**